### PR TITLE
OCP 4.14: IBM PowerVS IPI updates for PER enablement

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s): 4.14

Issue: https://issues.redhat.com/browse/OCPBUGS-20182

Link to docs preview:
https://65856--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_powervs/installing-ibm-cloud-account-power-vs#quotas-and-limits-ibm-power-vs_installing-ibm-cloud-account-power-vs
https://65856--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_powervs/installing-ibm-cloud-account-power-vs#installation-ibm-power-vs-regions_installing-ibm-cloud-account-power-vs

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/quotas-and-limits-ibm-power-vs.adoc [hunk 1]
```diff
@@ -32,6 +32,11 @@ VPC ALBs are supported. Classic ALBs are not supported for {ibmpowerProductName}
 
 There is a limit of two cloud connections per {ibmpowerProductName} Virtual Server instance. It is recommended that you have only one cloud connection in your {ibmpowerProductName} Virtual Server instance to serve your cluster.
 
+[NOTE]
+====
+Cloud Connections are no longer supported in `dal10`.
```
Comment (mjturek) [inline modules/quotas-and-limits-ibm-power-vs.adoc:37]: I might word it as
```
Cloud Connections are no longer supported in `dal10`. A transit gateway will be used instead.
```

**Review by mjturek — COMMENTED** [review]
One small addition

File: modules/quotas-and-limits-ibm-power-vs.adoc (hunk 1 above)
Comment (alishaIBM) [reply modules/quotas-and-limits-ibm-power-vs.adoc]: Ack. Updated the PR.
@mjturek

**Comment by alishaIBM** [comment]
@mjturek shall I go ahead with the peer review ?

**Comment by mjturek** [comment]
@alishaIBM yes please!

/lgtm

File: modules/installation-configuration-parameters.adoc [hunk 2]
```diff
@@ -1558,6 +1558,11 @@ Additional {ibmpowerProductName} Virtual Server configuration parameters are des
 1. Whether you define an existing resource group, or if the installer creates one, determines how the resource group is treated when the cluster is uninstalled. If you define a resource group, the installer removes all of the installer-provisioned resources, but leaves the resource group alone; if a resource group is created as part of the installation, the installer removes all of the installer provisioned resources and the resource group.
 2. To determine which profile best meets your needs, see https://cloud.ibm.com/docs/vpc?topic=vpc-profiles&interface=ui[Instance Profiles] in the IBM documentation.
 --
+
+[NOTE]
```
Comment (kcarmichael08) [inline modules/installation-configuration-parameters.adoc:1562]: I don't see a preview for this, but the text looks OK.

File: modules/installation-configuration-parameters.adoc [hunk 3]
```diff
@@ -1558,6 +1558,11 @@ Additional {ibmpowerProductName} Virtual Server configuration parameters are des
...
+====
+Cloud connections are no longer supported in the `install-config.yaml` while deploying in the `dal10` region as they have been replaced by the `Power Edge Router`.
```
Comment (kcarmichael08) [inline modules/installation-configuration-parameters.adoc:1564]:
```suggestion
Cloud connections are no longer supported in the `install-config.yaml` while deploying in the `dal10` region, as they have been replaced by the `Power Edge Router`.
```

Comment (kcarmichael08) [reply modules/installation-configuration-parameters.adoc:1564]: Making sure Power Edge Router is [supposed to be in backticks](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) (is an API object or a system item). I'm not sure what it is, so I thought I'd ask!

File: modules/quotas-and-limits-ibm-power-vs.adoc [hunk 4]
```diff
@@ -32,6 +32,11 @@ VPC ALBs are supported. Classic ALBs are not supported for {ibmpowerProductName}
 
 There is a limit of two cloud connections per {ibmpowerProductName} Virtual Server instance. It is recommended that you have only one cloud connection in your {ibmpowerProductName} Virtual Server instance to serve your cluster.
 
+[NOTE]
+====
+Cloud Connections are no longer supported in `dal10`. A transit gateway will be used instead.
```
Comment (kcarmichael08) [inline modules/quotas-and-limits-ibm-power-vs.adoc:37]:
```suggestion
Cloud connections are no longer supported in `dal10`. A transit gateway is used instead.
```

Comment (kcarmichael08) [reply modules/quotas-and-limits-ibm-power-vs.adoc:37]: Per IBM Style, present tense is preferred.

**Review by kcarmichael08 — COMMENTED** [review]
Few minor comments/questions; otherwise, lgtm!

File: modules/installation-configuration-parameters.adoc (hunk 2 above)
Comment (alishaIBM) [reply modules/installation-configuration-parameters.adoc:1562]: @kcarmichael08 here is the preview link for the NOTE: https://65856--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_powervs/installation-config-parameters-ibm-power-vs#installation-configuration-parameters-additional-ibm-cloud_installation-config-parameters-ibm-power-vs

File: modules/installation-configuration-parameters.adoc (hunk 3 above)
Comment (alishaIBM) [reply modules/installation-configuration-parameters.adoc]: @kcarmichael08 okay. I have removed the backticks. Thanks.

FYI: A Power Edge Router (PER) is a high-performance router that provides advanced routing capabilities for IBM® Power® Systems Virtual Server users. Reference: https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-per

**Comment by alishaIBM** [comment]
@kcarmichael08 thank you for the comments.
I have updated the PR. Request you to check. Thanks.

**Comment by kcarmichael08** [comment]
Hi @alishaIBM , it looks like there is a conflict that needs to be resolved before we can do the merge review.

**Comment by alishaIBM** [comment]
FYI : @kcarmichael08 @stevsmit

File: modules/installation-configuration-parameters.adoc [hunk 5]
```diff
@@ -657,6 +657,13 @@ ifdef::ibm-power-vs[]
 endif::ibm-power-vs[]
 |====
 
+ifdef::ibm-power-vs[]
+[NOTE]
+====
+Cloud connections are no longer supported in the `install-config.yaml` while deploying in the `dal10` region, as they have been replaced by the Power Edge Router (PER).
```
Comment (stevsmit) [inline modules/installation-configuration-parameters.adoc:663]: @alishaIBM This isn't enough to hold up a merge in my opinion, but we generally use a noun after specifying a YAML file. For example: 

```suggestion
Cloud connections are no longer supported in the `install-config.yaml` file while deploying in the `dal10` region, as they have been replaced by the Power Edge Router (PER).
```

**Pull request merged**